app/src/main/res/layout/profile_setup_house_rules.xml

```diff
@@ -9,152 +9,117 @@
-        app:layout_constraintVertical_bias="0.323" />
-
-    <TextView
-        android:id="@+id/guideline_1_text"
-        android:layout_width="332dp"
-        android:layout_height="48dp"
-        android:text="@string/make_sure_your_recordings_age_and_bio_are_true_to_who_you_are"
-        app:layout_constraintBottom_toTopOf="@+id/guideline_2"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.367"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/guideline_1"
-        app:layout_constraintVertical_bias="0.034" />
+        android:layout_above="@id/guideline_3"
+        android:layout_marginStart="@dimen/margin"
+        android:layout_marginEnd="@dimen/margin"
+        android:layout_marginBottom="@dimen/margin"
+        android:text="@string/do_not_be_too_quick_to_give_out_personal_information" />
 
     <TextView
         android:id="@+id/guideline_3"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_above="@id/guideline_3_text"
+        android:layout_marginStart="@dimen/margin"
         android:text="@string/u2713_play_it_cool"
-        android:textColor="#000000"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.088"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/guideline_2"
-        app:layout_constraintVertical_bias="0.215" />
+        android:textColor="#000000" />
 
     <TextView
         android:id="@+id/guideline_3_text"
-        android:layout_width="335dp"
-        android:layout_height="54dp"
-        android:text="@string/respect_others_and_treat_them_as_you_would_like_to_be_treated"
-        app:layout_constraintBottom_toTopOf="@+id/guideline_4"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.381"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/guideline_3"
-        app:layout_constraintVertical_bias="0.076" />
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_above="@id/guideline_4"
+        android:layout_marginStart="@dimen/margin"
+        android:layout_marginEnd="@dimen/margin"
+        android:layout_marginBottom="@dimen/margin"
+        android:text="@string/respect_others_and_treat_them_as_you_would_like_to_be_treated" />
+
+
+    <TextView
+        android:id="@+id/guideline_4"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_above="@id/guideline_4_text"
+        android:layout_marginStart="@dimen/margin"
+        android:text="@string/u2713_be_proactive"
+        android:textColor="#000000" />
```

Comment:
Because we should really tell something in reviews, but isn't #000 the default color anyway? Or maybe the theme has it defined somewhere else? I really won't care that much if you don't take my remark into account, it's not even you that put this line here originally haha.

Reply:
Actually I think we don't _have to_  write comments if there's nothing to say xD but indeed I think the default colors are changed in the theme xml of the app, to fit our theme

Reply:
Yeah sorry, it was in the beginning of the project. I just wanted something working and something uniform. Not a great concern I think but still we can change it to polish.

Reply:
Nothing to be sorry about, it actually turns grey otherwise x)